tests/unit/include/GoogleSync/GoogleSyncTest.php

```diff
@@ -8,7 +8,7 @@ class GoogleSyncTest extends \SuiteCRM\StateCheckerUnitAbstract
     protected $tester;
 
     /** @var ReflectionClass */
-    protected static $relection;
+    protected static $reflection;
```

Comment:
I like your reflection class solution to access protected and private things. :+1: